src/pages/404.js

```diff
@@ -1,12 +1,52 @@
 import React from 'react';
 import Layout from '../components/Layout';
+import styled from 'styled-components';
+import { Link } from 'gatsby';
+
+const Container = styled.div`
+  width: 50%;
+  min-height: 350px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  padding: 2.5rem 0;
+  margin: 0 auto;
+
+  text-align: center;
+`;
+
+const Title = styled.h1`
+  font-size: 72px;
+  font-weight: bold;
+  color: #1c376e;
+`;
+
+const Subtitle = styled.p`
+  font-size: 36px;
+  color: #1c376e;
+`;
 
 const NotFoundPage = () => (
   <Layout>
-    <div>
-      <h1>NOT FOUND</h1>
-      <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
-    </div>
+    <Container>
+      <Title>
+        4
+        <span
+          role="img"
+          aria-label="A yellow face with closed eyes wearing a white surgical mask, as used by health workers in hospitals. "
+        >
+          😷4
```

Comment:
I don't think you meant to have the 4 here

Reply:
Ops. Just fixed it